#request
string stream

---
#response
string streamUpdate


